docs/user/README.md

```diff
@@ -0,0 +1,62 @@
+
+# Kyma Telemetry Module
+
+Fundamentally, ["Observability"](https://opentelemetry.io/docs/concepts/observability-primer/) is a measure of how well the application's external outputs can reflect the internal states of single components. The insights that an application and the surrounding infrastructure expose are displayed in the form of metrics, traces, and logs - collectively, that's called "telemetry" or ["signals"](https://opentelemetry.io/docs/concepts/signals/). These can be exposed by employing modern instrumentation.
+
+In order to implement Day-2 operations for a distributed application running in a container runtime, the single components of an application must expose these signals by employing modern instrumentation. Furthermore, the signals must be collected and enriched with the infrastructural metadata in order to ship them to a target system.
+
+![Stages of Observability](./assets/general-stages.drawio.svg)
+
+There are plenty of observability backends available either as a service or as a self-manageable solution focusing on different aspects and scenarios. One approach will never fit all requirements, and the need to integrate with a specific backend will always be present. That's why Kyma supports you with instrumenting and shipping your telemetry data easily in a vendor-neutral way to enable observability for your application with little effort by integrating it into existing backends. These aspects must happen alongside your application, and managed tooling with guidance can benefit your initial investment and maintenance effort. Also, Kyma does not focus on providing a managed in-cluster backend solution, because a solution for an enterprise-grade setup demands a central out-cluster solution.
```

Comment:
Trying to rephrase it with less justification, more benefits:

```suggestion
Instead of providing a one-size-for-all backend solution, Kyma supports you with instrumenting and shipping your telemetry data in a vendor-neutral way. This way, you can conveniently enable observability for your application by integrating it into your existing or desired backends. Pick your favourite among many observability backends, available either as a service or as a self-manageable solution, that focus on different aspects and scenarios. If you configure your telemetry tooling alongside your application, managed tooling with guidance can benefit your initial investment and maintenance effort. An enterprise-grade setup demands a central solution outside the cluster, so we recommend in-cluster solutions only for testing purposes.
```